plugins/clean.py

```diff
@@ -17,31 +17,47 @@ def handle(self, directive, data):
 
     def _process_clean(self, targets):
         success = True
-        for target in targets:
-            success &= self._clean(target)
+        defaults = self._context.defaults().get(self._directive, {})
+        force = defaults.get('force', False)
+        for target in self._dispatch_targets(targets):
+            if isinstance(targets, dict):
+                force = targets[target].get('force', force)
+            success &= self._clean(target, force)
         if success:
             self._log.info('All targets have been cleaned')
         else:
             self._log.error('Some targets were not successfully cleaned')
         return success
 
-    def _clean(self, target):
+    def _clean(self, target, force):
         '''
-        Cleans all the broken symbolic links in target that point to
-        a subdirectory of the base directory.
+        Cleans all the broken symbolic links in target if that point to
```

Comment:
The grammar is slightly wrong here. Something like this would be better: "Cleans all the broken symbolic links in target if they point to a subdirectory of the base directory or if forced to clean."